charts/fluffy/Chart.yaml

```
@@ -0,0 +1,13 @@
+apiVersion: v2
+name: fluffy
+description: A Helm chart for running fluffy alongside LakeFS on Kubernetes
+type: application
+version: 0.1.0
+appVersion: "0.1.1"
```

Just to align the same style as "version"
```suggestion
appVersion: 0.1.1
```